crates/primitives/src/datalake/transactions/rlp_fields.rs

```diff
@@ -31,6 +31,8 @@ pub enum TransactionField {
     BlobVersionedHashes,
     // Only for EIP-4844 transactions
     MaxFeePerBlobGas,
+    // Sender address
+    Sender,
```

Comment:
This one we should confirm with @marcellobardus IMO. Doesnt really make sense to add, if the step limit makes it impossible to run.

Reply:
I recall that we synced with conclusion to have sender field. But we can revert it for now